pyannote/audio/labeling/models.py

```diff
@@ -231,3 +231,246 @@ def forward(self, sequences):
 
         elif self.task_type_ == TASK_REGRESSION:
             return torch.sigmoid(output)
+
+
+class ConvRNN(nn.Module):
```

Comment:
Would be nice to try and integrate this in the new `PyanNet` model as it is conceptually similar.

Reply:
Totally agree, but I would mark that as further improvements right ?

Reply:
Can you please add a reference to the paper from which you took inspiration at the end of the docstring?
Like what I already did for the [TristouNet](https://github.com/pyannote/pyannote-audio/blob/98fa2cb192bc0299e7444a90e00ba1633df8422a/pyannote/audio/embedding/models/tristounet.py#L59) model for instance.
```
    Reference
    ---------
    Hervé Bredin. "TristouNet: Triplet Loss for Speaker Turn Embedding."
    ICASSP 2017 (https://arxiv.org/abs/1609.04301)
```

Reply:
Solved